### DISCUSSION TITLE
Support beyond 2.5.x?

### DISCUSSION THREAD

**Discussion by cressman** [Cosium/spring-data-jpa-entity-graph#67] [General]
Thank you for this project. I have been very happy using it extensively in my projects. I am currently trying to upgrade my spring-data-jpa dependencies to 2.6 and noticed that there is no matching version in this project. From the testing I have done so far, I have not noticed any problems using spring-data-jpa-entity-graph 2.5 in combination with spring-data-jpa 2.6. Do I need a newer version? When will the next version be available?

Thanks so much!

**Comment by reda-alaoui** [comment]
Hello @cressman ,

The version is bumped on demand. I am still on 2.3 myself so I don't have the need for it myself. 
For each new major version, the plugin version and the spring-data version should be bumped to make sure all the tests are still green.
The version compability table of the README should be also updated.

Could you create a pull request for this?

**Reply by reda-alaoui** [reply]
I just released 2.6.0 to officially support Spring Data JPA 2.6.x.

**Reply by cressman** [reply]
Thank you!
